cluster_config/steps.yaml.tpl

```diff
@@ -26,12 +26,12 @@ Steps:
   HadoopJarStep:
     Args:
     - "spark-submit"
-    - "/opt/emr/generate_dataset_from_htme.py"
     - "--deploy-mode"
     - "cluster"
     - "--master"
     - "yarn"
     - "--conf"
     - "spark.yarn.submit.waitAppCompletion=true"
+    - "/opt/emr/generate_dataset_from_htme.py"
```

Comment:
this is what I was talking about earlier, this is better way of fixing unrecognised args issue